adoc/release-4.2.1.adoc

```diff
@@ -2,14 +2,51 @@
 
 === Deprecations in 4.2.1
 
+=== {Cilium} Update to 1.6.6 with Envoy
+
+* Updated Cilium from version 1.5.3 to version 1.6.6
+https://github.com/cilium/cilium/blob/v1.6.6/CHANGELOG.md
+* Provide Envoy-proxy support for Cilium
+* Envoy and its dependencies packaged for version 1.12.2
+* Cilium uses CRD and ConfigMap points on etcd are removed
+
+=== {Skuba}
+
+* Cilium 1.6.6 deployment with CRD
+
+=== Metrics Server
+    
+* Support monitoring of *CPU* and *memory* of a pod or node.
+    
+=== Cert Status Checker
+    
+* Exposes cluster-wide certificates status and use monitoring stack (Prometheus and Grafana) to receives alerts by Prometheus Alertmanager and monitors certificate status by Grafana dashboard.
+    
+=== VSphere VCP
+    
+* Allow Kubernetes pods to use VMWare vSphere Virtual Machine Disk (VMDK) volumes as persistent storage.
+
 === Required Actions
 
+* Migration from etcd to CRD in case of upgrade as described at https://docs.cilium.io/en/v1.6/install/upgrade/
```

Comment:
```suggestion
* Migration from etcd to CRD in case of upgrade as described in link:https://docs.cilium.io/en/v1.6/install/upgrade/[Upgrade Guide - Cilium]
```

Reply:
Does't skuba already take care of the migration?
Then what's the purpose of `Required Actions`?

Reply:
CC: @mmnelemane